### PR TITLE
Add deprecation note to README

### PR DESCRIPTION
**Pull request by DJMcNab**
See equivalent in Masonry

https://github.com/linebender/masonry/pull/60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +16,16 @@
...
 
+> [!Caution]
+> This README content has not been validated recently.
+>
+> Glazier is no longer supported, and should not be used.
+>
+> This repository is made available for reference purposes.
+>
+> Feel free to use the [#glazier channel](https://xi.zulipchat.com/#narrow/stream/351333-glazier) if reusing code from Glazier.
+> Some of the text input handling code is better supported than the equivalent in `winit`.
```
Comment (waywardmonkeys) [inline README.md:27]: Feels like "supported" is the wrong word here given "Glazier is no longer supported" ...

File: README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+> [!Important]
+> Linebender no longer uses Glazier, and so this project is no longer supported.
+>
+> You should prefer to use [winit](https://crates.io/crates/winit) instead.
```
Comment (waywardmonkeys) [inline README.md:4]: Should `winit` be in backticks here like it is below?

**Review by waywardmonkeys — APPROVED** [review]
LGTM apart from 2 minor nits that I accidentally submitted as single comments.

File: README.md (hunk 2 above)
Comment (DJMcNab) [reply README.md:4]: I could make a claim that one is talking about the crate whilst the other is talking about the project. But I've just removed the backticks

**Review by waywardmonkeys — APPROVED** [review]

**Pull request merged**